<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Authorization</class>
 <widget class="QDialog" name="Authorization">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1334</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="black_info">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
background-color: #18181B;
color: white;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>50</number>
      </property>
      <property name="topMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>50</number>
      </property>
      <property name="bottomMargin">
       <number>30</number>
      </property>
      <item alignment="Qt::AlignLeft">
       <widget class="QFrame" name="frame_3">
        <property name="styleSheet">
         <string notr="true">QFrame{
border: 0px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_app_img">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignLeft">
          <widget class="QLabel" name="label_app_name">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Название приложения</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_7">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>653</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_app_quote">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&quot;План - это половина успеха&quot;.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_app_author_quote">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Джордж Клюканов эль Вашингтон</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="registration_page">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_reg_grid">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame{
border: 0px;
color:black;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>50</number>
          </property>
          <property name="topMargin">
           <number>30</number>
          </property>
          <property name="rightMargin">
           <number>50</number>
          </property>
          <property name="bottomMargin">
           <number>30</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="4" column="1" colspan="2">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1" colspan="2">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1" rowspan="2" colspan="2">
           <widget class="QFrame" name="frame_reg_main">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label_reg_main">
               <property name="font">
                <font>
                 <pointsize>16</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Регистрация аккаунта</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Minimum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>11</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_reg_email">
               <property name="text">
                <string>Email</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_reg_email">
               <property name="placeholderText">
                <string>addres@example.com</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_reg_f">
               <property name="text">
                <string>Фамилия</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_reg_f">
               <property name="placeholderText">
                <string>Жмышенко</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_reg_n">
               <property name="text">
                <string>Имя</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_reg_n">
               <property name="placeholderText">
                <string>Валерий</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_reg_m">
               <property name="text">
                <string>Отчество</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_reg_m">
               <property name="placeholderText">
                <string>Альбертович</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_reg_url">
               <property name="text">
                <string>Адрес сервер Nextcloud</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_reg_url_nc">
               <property name="placeholderText">
                <string>url</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_reg_calendar_name">
               <property name="text">
                <string>Имя календаря</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_reg_calendar_name">
               <property name="placeholderText">
                <string>ExampleCalendar</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_reg_password1">
               <property name="text">
                <string>Пароль</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_reg_password1"/>
             </item>
             <item>
              <widget class="QLabel" name="label_reg_password2">
               <property name="text">
                <string>Подтвердите пароль</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_reg_password2"/>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QFrame" name="frame_haveacc_auth">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_haveacc">
                  <property name="text">
                   <string>Есть аккаунт?</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_haveacc_auth">
                  <property name="styleSheet">
                   <string notr="true">QPushButton{
	border: 0px;
	color:blue;
}

QPushButton:hover{
	text-decoration: underline;
	color:blue;
}</string>
                  </property>
                  <property name="text">
                   <string>Войти</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_reg">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QWidget{
background-color: #18181B;
color: white;
}</string>
               </property>
               <property name="text">
                <string>Регистрация</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0" colspan="4">
           <widget class="QFrame" name="frame_haveacc_auth_up">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="leftMargin">
              <number>30</number>
             </property>
             <item alignment="Qt::AlignRight">
              <widget class="QPushButton" name="pushButton_haveacc_auth_up">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	border: 0px;
	color:black;
}

QPushButton:hover{
	text-decoration: underline;
	color:black;
}</string>
               </property>
               <property name="text">
                <string>Вход</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="0" rowspan="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="3" rowspan="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="authorization_page">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_auth_grid">
         <property name="styleSheet">
          <string notr="true">QFrame{
border: 0px;
color:black;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>50</number>
          </property>
          <property name="topMargin">
           <number>30</number>
          </property>
          <property name="rightMargin">
           <number>50</number>
          </property>
          <property name="bottomMargin">
           <number>30</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="2" column="0">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>150</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QFrame" name="auth_frame_main">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="label_auth_login">
               <property name="font">
                <font>
                 <pointsize>16</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Вход в аккаунт</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_6">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Minimum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>11</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_auth_f">
               <property name="text">
                <string>Фамилия</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_auth_f">
               <property name="text">
                <string/>
               </property>
               <property name="placeholderText">
                <string>Жмышенко</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_auth_n">
               <property name="text">
                <string>Имя</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_auth_n">
               <property name="text">
                <string/>
               </property>
               <property name="placeholderText">
                <string>Валерий</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_auth_m">
               <property name="text">
                <string>Отчество</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_auth_m">
               <property name="text">
                <string/>
               </property>
               <property name="placeholderText">
                <string>Альбертович</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_auth_email">
               <property name="text">
                <string>Почта</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_auth_email">
               <property name="placeholderText">
                <string>address@example.com</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_auth_code">
               <property name="text">
                <string>Шифр входа</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_auth_code"/>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QFrame" name="reg_frame">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_noacc">
                  <property name="text">
                   <string>Нет аккаунта?</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="label_noacc_reg">
                  <property name="styleSheet">
                   <string notr="true">QPushButton{
	border: 0px;
	color:blue;
}

QPushButton:hover{
	text-decoration: underline;
	color:blue;
}</string>
                  </property>
                  <property name="text">
                   <string>Регистрация</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_auth">
               <property name="styleSheet">
                <string notr="true">QWidget{
background-color: #18181B;
color: white;
}</string>
               </property>
               <property name="text">
                <string>Вход в аккаунт</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="3">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>150</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1" colspan="2">
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="1" colspan="2">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0" colspan="4" alignment="Qt::AlignRight">
           <widget class="QFrame" name="frame_noacc_reg_up">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="pushButton_noacc_reg_up">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	border: 0px;
	color:black;
}

QPushButton:hover{
	text-decoration: underline;
	color:black;
}</string>
               </property>
               <property name="text">
                <string>Регистрация</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
